From edcce188ea0e8ac413557a945a4e761b32bb0d06 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 16 Apr 2020 07:46:31 -0500
Subject: [PATCH 17/17] arm64: dts: beacon-imx8mm-kit: Fix GPIO expander
 compatible flag

The gpio driver has been updated to suppor the pull-up resistors,
but the compatible flag needs to reflect the hardware can support it.

This patch updates the compatible flag for the GPIO expander to
support the nxp,pcal6416 which supports pull-up resistors.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
index 26188ea26ef4..bd91dd55064b 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
@@ -375,7 +375,7 @@
 	status = "okay";
 
 	pca6416_0: gpio@20 {
-		compatible = "ti,tca6416";
+		compatible = "nxp,pcal6416";
 		reg = <0x20>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_pcal6414>;
@@ -386,7 +386,7 @@
 	};
 
 	pca6416_1: gpio@21 {
-		compatible = "ti,tca6416";
+		compatible = "nxp,pcal6416";
 		reg = <0x21>;
 		gpio-controller;
 		#gpio-cells = <2>;
-- 
2.17.1

